docs/openj9_support.md

```diff
@@ -282,4 +323,17 @@ The project builds and tests OpenJDK with OpenJ9 on a number of platforms. The o
 | Windows x86 64-bit            | Windows Server 2019    | Microsoft Visual Studio 2022          |
 | macOS x86 64-bit              | macOS 10.15.7          | xcode 12.4 and clang 12.0.0           |
 | macOS AArch64                 | macOS 11.5.2           | xcode 13.0 and clang 13.0.0           |
-| AIX POWER BE 64-bit           | AIX 7.2 TL5            | xlc/C++ 16.1.0.20                    |
+| AIX POWER BE 64-bit           | AIX 7.2 TL5            | xlc/C++ 16.1.0.20                     |
+
+### OpenJDK 25 and later
+
+| Platform                      | Operating system       | Compiler                              |
+|-------------------------------|------------------------|---------------------------------------|
+| Linux x86 64-bit              | CentOS 7.9             | gcc 13.2                              |
+| Linux on POWER LE 64-bit      | CentOS 7.9             | gcc 13.2                              |
+| Linux on IBM Z 64-bit         | RHEL 7.9               | gcc 13.2                              |
+| Linux AArch64 64-bit          | CentOS 7.9             | gcc 10.3                              |
+| Windows x86 64-bit            | Windows Server 2019    | Microsoft Visual Studio 2022          |
+| macOS x86 64-bit              | macOS 10.15.7          | xcode 12.4 and clang 12.0.0           |
+| macOS AArch64                 | macOS 11.5.2           | xcode 13.0 and clang 13.0.0           |
+| AIX POWER BE 64-bit           | AIX 7.2 TL5            | xlc/C++ 17.1.3.0                   |
```

Comment:
This needs to stay as 16.1.0.20 for the 0.55 release. We expect to change it in a later release.